core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -191,7 +191,7 @@ class AdminManager(val config: KafkaConfig,
   def deleteTopics(timeout: Int,
                    topics: Set[String],
                    responseCallback: Map[String, Errors] => Unit): Unit = {
-
+    info(s"Deleting topics ${topics.mkString(", ")}")
```

Comment:
The other methods in this class don't seem to have a log line like this one. Seems like we should be consistent one way or another.

Reply:
I think that we should have logs for operations that update resources. Adding logs to `createPartitions()` and `createTopics()` sounds good to me, as long as they aren't overly verbose.

For incrementalAlterConfigs/alterConfigs, I'm not sure